### DISCUSSION TITLE
Any chance for an individual offline version?

### DISCUSSION THREAD

**Discussion by ghost** [Leantime/leantime#2214] [General]
I've tried many project management tools recently, but I didn't like any. Then I found Leantime via AlternativeTo.net, and it seems like exactly what I've been looking for. I really want to try this, but I'm not good at tech, and I don't understand Docker. I'm not part of a startup or small team either.

So, any chance you could make an individual offline version? Like a simple exe file I can install on Windows 10. I don't want my data on any server, or always need to be online.

Or do I fundamentally misunderstand this project?

You mentioned that you designed it with ADHD and autism in mind, which is a double match for me! (lol)

**Comment by marcelfolaron** [comment]
Thanks for reaching out. Unfortunately it is not so easy to have an individual offline version available since Leantime was purposefully built to be set up on a server. 

You have 2 options here: You go with one of our cloud plans (https://leantime.io). Each customer gets their own database and space so all customer data is strictly separated. We're always happy to provide data exports. 

Alternatively (and depending on your budget) we can host a dedicated instance of Leantime on a server of your choice.
